frontend/dockerfile/dockerfile_test.go

```diff
@@ -3886,7 +3886,8 @@ ONBUILD RUN mkdir -p /out && echo -n 11 >> /out/foo
 }
 
 func testCacheMultiPlatformImportExport(t *testing.T, sb integration.Sandbox) {
-	integration.SkipIfDockerd(t, sb, "direct push")
+	integration.SkipIfDockerdMoby(t, sb, "direct push")
+	integration.SkipIfDockerdSnapshotter(t, sb, "cache multi-platform hangs")
```

Comment:
This one hangs with `containerd-snapshotter` which might be because docker daemon panics. Will repro and post the logs.

Reply:
Ok this one fails here: https://github.com/moby/buildkit/blob/c717d6aa7543d4b83395e0552ef2eb311f563aab/frontend/dockerfile/dockerfile_test.go#L3996-L3997

Looking at the logs it seems it fails to pull the frontend gateway image:

```
=== RUN   TestIntegration
time="2022-10-18T12:47:34Z" level=info msg="trying next host - response was http.StatusNotFound" host="localhost:33377"
    run.go:251: copied local:/mainline.tar to local mirror localhost:33377/buildkit_test/pnkfrjl08i2qpegzaj4mvxww6:latest
=== RUN   TestIntegration/TestCacheMultiPlatformImportExport/worker=dockerd-containerd/frontend=gateway
=== PAUSE TestIntegration/TestCacheMultiPlatformImportExport/worker=dockerd-containerd/frontend=gateway
=== CONT  TestIntegration/TestCacheMultiPlatformImportExport/worker=dockerd-containerd/frontend=gateway
> startCmd 2022-10-18 12:47:38.71032318 +0000 UTC m=+4.237854474 [/usr/bin/dockerd --config-file /tmp/integration4180081475/daemon.json --data-root /tmp/integration4180081475/dww2uhqidy16j/root --exec-root /tmp/dxr/dww2uhqidy16j --pidfile /tmp/integration4180081475/dww2uhqidy16j/docker.pid --host unix:///tmp/integration4180081475/dww2uhqidy16j/docker.sock --userland-proxy=false --containerd-namespace dww2uhqidy16j --containerd-plugins-namespace dww2uhqidy16jp --bip 10.66.66.1/24 --default-address-pool base=10.66.66.0/16,size=24 --debug]
time="2022-10-18T12:47:38.731911080Z" level=info msg="Starting up"
time="2022-10-18T12:47:38.732996153Z" level=warning msg="could not change group /tmp/integration4180081475/dww2uhqidy16j/docker.sock to docker: group docker not found"
time="2022-10-18T12:47:38.733088135Z" level=debug msg="Listener created for HTTP on unix (/tmp/integration4180081475/dww2uhqidy16j/docker.sock)"
time="2022-10-18T12:47:38.733112451Z" level=debug msg="Containerd not running, starting daemon managed containerd"
time="2022-10-18T12:47:38.733508282Z" level=info msg="libcontainerd: started new containerd process" pid=630
time="2022-10-18T12:47:38.733539921Z" level=info msg="[core] original dial target is: \"unix:///tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\"" module=grpc
time="2022-10-18T12:47:38.733559378Z" level=info msg="[core] parsed dial target is: {Scheme:unix Authority: Endpoint:tmp/dxr/dww2uhqidy16j/containerd/containerd.sock URL:{Scheme:unix Opaque: User: Host: Path:/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock RawPath: ForceQuery:false RawQuery: Fragment: RawFragment:}}" module=grpc
time="2022-10-18T12:47:38.733574366Z" level=info msg="[core] Channel authority set to \"localhost\"" module=grpc
time="2022-10-18T12:47:38.733585086Z" level=info msg="[core] ccResolverWrapper: sending update to cc: {[{/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock  0xc000010b48 <nil> 0 <nil>}] <nil> <nil>}" module=grpc
time="2022-10-18T12:47:38.733599062Z" level=info msg="[core] ClientConn switching balancer to \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.733602719Z" level=info msg="[core] Channel switches to new LB policy \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.733648545Z" level=info msg="[core] Subchannel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.733681617Z" level=info msg="[core] Subchannel picks a new address \"/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\" to connect" module=grpc
time="2022-10-18T12:47:38.733728345Z" level=info msg="[core] Channel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.752662021Z" level=info msg="starting containerd" revision=10c12954828e7c7c9b6e0ea9b0c02b01407d3ae1 version=v1.6.6
time="2022-10-18T12:47:38.760085667Z" level=info msg="loading plugin \"io.containerd.content.v1.content\"..." type=io.containerd.content.v1
time="2022-10-18T12:47:38.760246839Z" level=info msg="loading plugin \"io.containerd.snapshotter.v1.aufs\"..." type=io.containerd.snapshotter.v1
time="2022-10-18T12:47:38.761226534Z" level=info msg="skip loading plugin \"io.containerd.snapshotter.v1.aufs\"..." error="aufs is not supported (modprobe aufs failed: exit status 1 \"modprobe: can't change directory to '/lib/modules': No such file or directory\\n\"): skip plugin" type=io.containerd.snapshotter.v1
time="2022-10-18T12:47:38.761250278Z" level=info msg="loading plugin \"io.containerd.snapshotter.v1.devmapper\"..." type=io.containerd.snapshotter.v1
time="2022-10-18T12:47:38.761259866Z" level=warning msg="failed to load plugin io.containerd.snapshotter.v1.devmapper" error="devmapper not configured"
time="2022-10-18T12:47:38.761264885Z" level=info msg="loading plugin \"io.containerd.snapshotter.v1.native\"..." type=io.containerd.snapshotter.v1
time="2022-10-18T12:47:38.761359903Z" level=info msg="loading plugin \"io.containerd.snapshotter.v1.overlayfs\"..." type=io.containerd.snapshotter.v1
time="2022-10-18T12:47:38.761568835Z" level=info msg="loading plugin \"io.containerd.snapshotter.v1.zfs\"..." type=io.containerd.snapshotter.v1
time="2022-10-18T12:47:38.761754062Z" level=info msg="skip loading plugin \"io.containerd.snapshotter.v1.zfs\"..." error="path /tmp/integration4180081475/dww2uhqidy16j/root/containerd/daemon/io.containerd.snapshotter.v1.zfs must be a zfs filesystem to be used with the zfs snapshotter: skip plugin" type=io.containerd.snapshotter.v1
time="2022-10-18T12:47:38.761773839Z" level=info msg="loading plugin \"io.containerd.metadata.v1.bolt\"..." type=io.containerd.metadata.v1
time="2022-10-18T12:47:38.761829794Z" level=warning msg="could not use snapshotter devmapper in metadata plugin" error="devmapper not configured"
time="2022-10-18T12:47:38.761846605Z" level=info msg="metadata content store policy set" policy=shared
time="2022-10-18T12:47:38.776891868Z" level=info msg="loading plugin \"io.containerd.differ.v1.walking\"..." type=io.containerd.differ.v1
time="2022-10-18T12:47:38.776942493Z" level=info msg="loading plugin \"io.containerd.event.v1.exchange\"..." type=io.containerd.event.v1
time="2022-10-18T12:47:38.776953664Z" level=info msg="loading plugin \"io.containerd.gc.v1.scheduler\"..." type=io.containerd.gc.v1
time="2022-10-18T12:47:38.777002515Z" level=info msg="loading plugin \"io.containerd.service.v1.introspection-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777023906Z" level=info msg="loading plugin \"io.containerd.service.v1.containers-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777033834Z" level=info msg="loading plugin \"io.containerd.service.v1.content-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777047420Z" level=info msg="loading plugin \"io.containerd.service.v1.diff-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777069651Z" level=info msg="loading plugin \"io.containerd.service.v1.images-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777080482Z" level=info msg="loading plugin \"io.containerd.service.v1.leases-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777089198Z" level=info msg="loading plugin \"io.containerd.service.v1.namespaces-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777096822Z" level=info msg="loading plugin \"io.containerd.service.v1.snapshots-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.777103915Z" level=info msg="loading plugin \"io.containerd.runtime.v1.linux\"..." type=io.containerd.runtime.v1
time="2022-10-18T12:47:38.777278152Z" level=info msg="loading plugin \"io.containerd.runtime.v2.task\"..." type=io.containerd.runtime.v2
time="2022-10-18T12:47:38.777387086Z" level=info msg="loading plugin \"io.containerd.monitor.v1.cgroups\"..." type=io.containerd.monitor.v1
time="2022-10-18T12:47:38.777830106Z" level=info msg="loading plugin \"io.containerd.service.v1.tasks-service\"..." type=io.containerd.service.v1
time="2022-10-18T12:47:38.778166797Z" level=debug msg="No RDT config file specified, RDT not configured"
time="2022-10-18T12:47:38.778320044Z" level=info msg="loading plugin \"io.containerd.grpc.v1.introspection\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778343338Z" level=info msg="loading plugin \"io.containerd.internal.v1.restart\"..." type=io.containerd.internal.v1
time="2022-10-18T12:47:38.778424259Z" level=info msg="loading plugin \"io.containerd.grpc.v1.containers\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778446771Z" level=info msg="loading plugin \"io.containerd.grpc.v1.content\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778456410Z" level=info msg="loading plugin \"io.containerd.grpc.v1.diff\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778467059Z" level=info msg="loading plugin \"io.containerd.grpc.v1.events\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778477168Z" level=info msg="loading plugin \"io.containerd.grpc.v1.healthcheck\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778495663Z" level=info msg="loading plugin \"io.containerd.grpc.v1.images\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778506523Z" level=info msg="loading plugin \"io.containerd.grpc.v1.leases\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778515731Z" level=info msg="loading plugin \"io.containerd.grpc.v1.namespaces\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778524928Z" level=info msg="loading plugin \"io.containerd.internal.v1.opt\"..." type=io.containerd.internal.v1
time="2022-10-18T12:47:38.778892957Z" level=info msg="loading plugin \"io.containerd.grpc.v1.snapshots\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778924787Z" level=info msg="loading plugin \"io.containerd.grpc.v1.tasks\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778932571Z" level=info msg="loading plugin \"io.containerd.grpc.v1.version\"..." type=io.containerd.grpc.v1
time="2022-10-18T12:47:38.778939054Z" level=info msg="loading plugin \"io.containerd.tracing.processor.v1.otlp\"..." type=io.containerd.tracing.processor.v1
time="2022-10-18T12:47:38.778948551Z" level=info msg="skip loading plugin \"io.containerd.tracing.processor.v1.otlp\"..." error="no OpenTelemetry endpoint: skip plugin" type=io.containerd.tracing.processor.v1
time="2022-10-18T12:47:38.778963319Z" level=info msg="loading plugin \"io.containerd.internal.v1.tracing\"..." type=io.containerd.internal.v1
time="2022-10-18T12:47:38.778976874Z" level=error msg="failed to initialize a tracing processor \"otlp\"" error="no OpenTelemetry endpoint: skip plugin"
time="2022-10-18T12:47:38.779189733Z" level=info msg=serving... address=/tmp/dxr/dww2uhqidy16j/containerd/containerd-debug.sock
time="2022-10-18T12:47:38.779240999Z" level=info msg=serving... address=/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock.ttrpc
time="2022-10-18T12:47:38.779282707Z" level=info msg=serving... address=/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock
time="2022-10-18T12:47:38.779310960Z" level=debug msg="sd notification" error="<nil>" notified=false state="READY=1"
time="2022-10-18T12:47:38.779318615Z" level=info msg="containerd successfully booted in 0.027415s"
time="2022-10-18T12:47:38.785763808Z" level=info msg="[core] Subchannel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:38.785822549Z" level=info msg="[core] Channel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:38.785842516Z" level=debug msg="Created containerd monitoring client" address=/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock
time="2022-10-18T12:47:38.786499586Z" level=debug msg="Started daemon managed containerd"
time="2022-10-18T12:47:38.786997078Z" level=debug msg="Golang's threads limit set to 230760"
time="2022-10-18T12:47:38.787299335Z" level=info msg="[core] original dial target is: \"unix:///tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\"" module=grpc
time="2022-10-18T12:47:38.787327598Z" level=info msg="[core] parsed dial target is: {Scheme:unix Authority: Endpoint:tmp/dxr/dww2uhqidy16j/containerd/containerd.sock URL:{Scheme:unix Opaque: User: Host: Path:/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock RawPath: ForceQuery:false RawQuery: Fragment: RawFragment:}}" module=grpc
time="2022-10-18T12:47:38.787336224Z" level=info msg="[core] Channel authority set to \"localhost\"" module=grpc
time="2022-10-18T12:47:38.787347865Z" level=info msg="[core] ccResolverWrapper: sending update to cc: {[{/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock  0xc000690a40 <nil> 0 <nil>}] <nil> <nil>}" module=grpc
time="2022-10-18T12:47:38.787362844Z" level=info msg="[core] ClientConn switching balancer to \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.787367472Z" level=info msg="[core] Channel switches to new LB policy \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.787385406Z" level=debug msg="metrics API listening on /tmp/dxr/dww2uhqidy16j/metrics.sock"
time="2022-10-18T12:47:38.787388592Z" level=info msg="[core] Subchannel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.787433837Z" level=info msg="[core] Subchannel picks a new address \"/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\" to connect" module=grpc
time="2022-10-18T12:47:38.787513650Z" level=info msg="[core] Channel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.787594247Z" level=info msg="[core] Subchannel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:38.787617671Z" level=info msg="[core] Channel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:38.788929879Z" level=info msg="[core] original dial target is: \"unix:///tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\"" module=grpc
time="2022-10-18T12:47:38.788967499Z" level=info msg="[core] parsed dial target is: {Scheme:unix Authority: Endpoint:tmp/dxr/dww2uhqidy16j/containerd/containerd.sock URL:{Scheme:unix Opaque: User: Host: Path:/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock RawPath: ForceQuery:false RawQuery: Fragment: RawFragment:}}" module=grpc
time="2022-10-18T12:47:38.788975424Z" level=info msg="[core] Channel authority set to \"localhost\"" module=grpc
time="2022-10-18T12:47:38.788984711Z" level=info msg="[core] ccResolverWrapper: sending update to cc: {[{/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock  0xc0005c2370 <nil> 0 <nil>}] <nil> <nil>}" module=grpc
time="2022-10-18T12:47:38.788997666Z" level=info msg="[core] ClientConn switching balancer to \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.789001503Z" level=info msg="[core] Channel switches to new LB policy \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.789029004Z" level=info msg="[core] Subchannel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.789048441Z" level=info msg="[core] Subchannel picks a new address \"/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\" to connect" module=grpc
time="2022-10-18T12:47:38.789108253Z" level=info msg="[core] Channel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.789278522Z" level=info msg="[core] Subchannel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:38.789307055Z" level=info msg="[core] Channel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:38.789705302Z" level=debug msg="processing event stream" module=libcontainerd namespace=dww2uhqidy16jp
time="2022-10-18T12:47:38.789727242Z" level=debug msg="Using default logging driver json-file"
time="2022-10-18T12:47:38.789880459Z" level=debug msg="No quota support for local volumes in /tmp/integration4180081475/dww2uhqidy16j/root/volumes: Filesystem does not support, or has not enabled quotas"
time="2022-10-18T12:47:38.813318293Z" level=info msg="Loading containers: start."
time="2022-10-18T12:47:38.813365030Z" level=debug msg="Option Experimental: false"
time="2022-10-18T12:47:38.813371763Z" level=debug msg="Option DefaultDriver: bridge"
time="2022-10-18T12:47:38.813374959Z" level=debug msg="Option DefaultNetwork: bridge"
time="2022-10-18T12:47:38.813378666Z" level=debug msg="Network Control Plane MTU: 1500"
time="2022-10-18T12:47:38.813490096Z" level=debug msg="processing event stream" module=libcontainerd namespace=dww2uhqidy16j
time="2022-10-18T12:47:38.814548307Z" level=debug msg="Fail to initialize firewalld: Failed to connect to D-Bus system bus: dial unix /var/run/dbus/system_bus_socket: connect: no such file or directory, using raw iptables instead"
time="2022-10-18T12:47:38.815404071Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -j DOCKER-ISOLATION]"
time="2022-10-18T12:47:38.815800412Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -D PREROUTING -m addrtype --dst-type LOCAL -j DOCKER]"
time="2022-10-18T12:47:38.816256488Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -D OUTPUT -m addrtype --dst-type LOCAL ! --dst 127.0.0.0/8 -j DOCKER]"
time="2022-10-18T12:47:38.816731006Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -D OUTPUT -m addrtype --dst-type LOCAL -j DOCKER]"
time="2022-10-18T12:47:38.817117009Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -D PREROUTING]"
time="2022-10-18T12:47:38.817572311Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -D OUTPUT]"
time="2022-10-18T12:47:38.818009036Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -F DOCKER]"
time="2022-10-18T12:47:38.818367923Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -X DOCKER]"
time="2022-10-18T12:47:38.818685287Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -F DOCKER]"
time="2022-10-18T12:47:38.819026525Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -X DOCKER]"
time="2022-10-18T12:47:38.819360742Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -F DOCKER-ISOLATION-STAGE-1]"
time="2022-10-18T12:47:38.819674219Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -X DOCKER-ISOLATION-STAGE-1]"
time="2022-10-18T12:47:38.820024035Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -F DOCKER-ISOLATION-STAGE-2]"
time="2022-10-18T12:47:38.820334076Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -X DOCKER-ISOLATION-STAGE-2]"
time="2022-10-18T12:47:38.820678911Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -F DOCKER-ISOLATION]"
time="2022-10-18T12:47:38.820991556Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -X DOCKER-ISOLATION]"
time="2022-10-18T12:47:38.821303792Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -n -L DOCKER]"
time="2022-10-18T12:47:38.821628100Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -N DOCKER]"
time="2022-10-18T12:47:38.822034065Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -n -L DOCKER]"
time="2022-10-18T12:47:38.822387341Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -N DOCKER]"
time="2022-10-18T12:47:38.822729001Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -n -L DOCKER-ISOLATION-STAGE-1]"
time="2022-10-18T12:47:38.823132867Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -N DOCKER-ISOLATION-STAGE-1]"
time="2022-10-18T12:47:38.823470441Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -n -L DOCKER-ISOLATION-STAGE-2]"
time="2022-10-18T12:47:38.823781533Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -N DOCKER-ISOLATION-STAGE-2]"
time="2022-10-18T12:47:38.824117893Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C DOCKER-ISOLATION-STAGE-1 -j RETURN]"
time="2022-10-18T12:47:38.824660269Z" level=debug msg="/usr/bin/iptables, [--wait -A DOCKER-ISOLATION-STAGE-1 -j RETURN]"
time="2022-10-18T12:47:38.825016666Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C DOCKER-ISOLATION-STAGE-2 -j RETURN]"
time="2022-10-18T12:47:38.825330695Z" level=debug msg="/usr/bin/iptables, [--wait -A DOCKER-ISOLATION-STAGE-2 -j RETURN]"
time="2022-10-18T12:47:38.844845731Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -n -L DOCKER-USER]"
time="2022-10-18T12:47:38.845381094Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -N DOCKER-USER]"
time="2022-10-18T12:47:38.846044012Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C DOCKER-USER -j RETURN]"
time="2022-10-18T12:47:38.846775626Z" level=debug msg="/usr/bin/iptables, [--wait -A DOCKER-USER -j RETURN]"
time="2022-10-18T12:47:38.847204174Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.847606292Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.866431305Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -n -L DOCKER-USER]"
time="2022-10-18T12:47:38.866930782Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C DOCKER-USER -j RETURN]"
time="2022-10-18T12:47:38.867393428Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.867755225Z" level=debug msg="/usr/bin/iptables, [--wait -D FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.868164236Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.869111727Z" level=debug msg="Allocating IPv4 pools for network bridge (13d5075f26ed196b43a0588832a286eee02a1df8fb92e34d640697788d014238)"
time="2022-10-18T12:47:38.869138958Z" level=debug msg="RequestPool(LocalDefault, 10.66.66.0/24, , map[], false)"
time="2022-10-18T12:47:38.869156220Z" level=debug msg="RequestAddress(LocalDefault/10.66.66.0/24, 10.66.66.1, map[RequestAddressType:com.docker.network.gateway])"
time="2022-10-18T12:47:38.869165317Z" level=debug msg="Request address PoolID:10.66.66.0/24 App: ipam/default/data, ID: LocalDefault/10.66.66.0/24, DBIndex: 0x0, Bits: 256, Unselected: 254, Sequence: (0x80000000, 1)->(0x0, 6)->(0x1, 1)->end Curr:0 Serial:false PrefAddress:10.66.66.1 "
time="2022-10-18T12:47:38.869199161Z" level=debug msg="Did not find any interface with name docker0: Link not found"
time="2022-10-18T12:47:38.869253142Z" level=debug msg="Setting bridge mac address to 02:42:33:1f:b1:c3"
time="2022-10-18T12:47:38.869513910Z" level=debug msg="Assigning address to bridge interface docker0: 10.66.66.1/24"
time="2022-10-18T12:47:38.869601464Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -C POSTROUTING -s 10.66.66.0/24 ! -o docker0 -j MASQUERADE]"
time="2022-10-18T12:47:38.870035476Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -I POSTROUTING -s 10.66.66.0/24 ! -o docker0 -j MASQUERADE]"
time="2022-10-18T12:47:38.870426119Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -C POSTROUTING -m addrtype --src-type LOCAL -o docker0 -j MASQUERADE]"
time="2022-10-18T12:47:38.870835156Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -I POSTROUTING -m addrtype --src-type LOCAL -o docker0 -j MASQUERADE]"
time="2022-10-18T12:47:38.871240745Z" level=debug msg="/usr/bin/iptables, [--wait -D FORWARD -i docker0 -o docker0 -j DROP]"
time="2022-10-18T12:47:38.871608894Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -i docker0 -o docker0 -j ACCEPT]"
time="2022-10-18T12:47:38.871943692Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -i docker0 -o docker0 -j ACCEPT]"
time="2022-10-18T12:47:38.872292395Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -i docker0 ! -o docker0 -j ACCEPT]"
time="2022-10-18T12:47:38.872621802Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -i docker0 ! -o docker0 -j ACCEPT]"
time="2022-10-18T12:47:38.872963993Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -C PREROUTING -m addrtype --dst-type LOCAL -j DOCKER]"
time="2022-10-18T12:47:38.873336371Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -A PREROUTING -m addrtype --dst-type LOCAL -j DOCKER]"
time="2022-10-18T12:47:38.873724979Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -C OUTPUT -m addrtype --dst-type LOCAL -j DOCKER]"
time="2022-10-18T12:47:38.874141198Z" level=debug msg="/usr/bin/iptables, [--wait -t nat -A OUTPUT -m addrtype --dst-type LOCAL -j DOCKER]"
time="2022-10-18T12:47:38.874528614Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -o docker0 -j DOCKER]"
time="2022-10-18T12:47:38.874843333Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -o docker0 -j DOCKER]"
time="2022-10-18T12:47:38.875209279Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -o docker0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT]"
time="2022-10-18T12:47:38.875665984Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -o docker0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT]"
time="2022-10-18T12:47:38.876092573Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -j DOCKER-ISOLATION-STAGE-1]"
time="2022-10-18T12:47:38.876430857Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -j DOCKER-ISOLATION-STAGE-1]"
time="2022-10-18T12:47:38.876768640Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C DOCKER-ISOLATION-STAGE-1 -i docker0 ! -o docker0 -j DOCKER-ISOLATION-STAGE-2]"
time="2022-10-18T12:47:38.877237898Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -I DOCKER-ISOLATION-STAGE-1 -i docker0 ! -o docker0 -j DOCKER-ISOLATION-STAGE-2]"
time="2022-10-18T12:47:38.877678234Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C DOCKER-ISOLATION-STAGE-2 -o docker0 -j DROP]"
time="2022-10-18T12:47:38.878028039Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -I DOCKER-ISOLATION-STAGE-2 -o docker0 -j DROP]"
time="2022-10-18T12:47:38.883938240Z" level=debug msg="garbage collected" d=5.949395ms
time="2022-10-18T12:47:38.899967726Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -n -L DOCKER-USER]"
time="2022-10-18T12:47:38.900610481Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C DOCKER-USER -j RETURN]"
time="2022-10-18T12:47:38.901066445Z" level=debug msg="/usr/bin/iptables, [--wait -t filter -C FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.901445285Z" level=debug msg="/usr/bin/iptables, [--wait -D FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.901860102Z" level=debug msg="/usr/bin/iptables, [--wait -I FORWARD -j DOCKER-USER]"
time="2022-10-18T12:47:38.902387640Z" level=info msg="Loading containers: done."
time="2022-10-18T12:47:38.907452557Z" level=warning msg="failed to retrieve docker-init version: exec: \"docker-init\": executable file not found in $PATH"
time="2022-10-18T12:47:38.908018237Z" level=warning msg="WARNING: No blkio throttle.read_bps_device support"
time="2022-10-18T12:47:38.908039467Z" level=warning msg="WARNING: No blkio throttle.write_bps_device support"
time="2022-10-18T12:47:38.908043495Z" level=warning msg="WARNING: No blkio throttle.read_iops_device support"
time="2022-10-18T12:47:38.908046220Z" level=warning msg="WARNING: No blkio throttle.write_iops_device support"
time="2022-10-18T12:47:38.908059996Z" level=info msg="Docker daemon" commit=d534d16dfe graphdriver=overlayfs version=dev
time="2022-10-18T12:47:38.908309854Z" level=info msg="Daemon has completed initialization"
time="2022-10-18T12:47:38.922962231Z" level=info msg="[core] original dial target is: \"unix:///tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\"" module=grpc
time="2022-10-18T12:47:38.922995163Z" level=info msg="[core] parsed dial target is: {Scheme:unix Authority: Endpoint:tmp/dxr/dww2uhqidy16j/containerd/containerd.sock URL:{Scheme:unix Opaque: User: Host: Path:/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock RawPath: ForceQuery:false RawQuery: Fragment: RawFragment:}}" module=grpc
time="2022-10-18T12:47:38.923003488Z" level=info msg="[core] Channel authority set to \"localhost\"" module=grpc
time="2022-10-18T12:47:38.923017815Z" level=info msg="[core] ccResolverWrapper: sending update to cc: {[{/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock  0xc0005c2250 <nil> 0 <nil>}] <nil> <nil>}" module=grpc
time="2022-10-18T12:47:38.923034897Z" level=info msg="[core] ClientConn switching balancer to \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.923040397Z" level=info msg="[core] Channel switches to new LB policy \"pick_first\"" module=grpc
time="2022-10-18T12:47:38.923095481Z" level=info msg="[core] Subchannel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.923133933Z" level=info msg="[core] Subchannel picks a new address \"/tmp/dxr/dww2uhqidy16j/containerd/containerd.sock\" to connect" module=grpc
time="2022-10-18T12:47:38.923169528Z" level=info msg="[core] Channel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:38.923422973Z" level=info msg="[core] Subchannel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:38.923467507Z" level=info msg="[core] Channel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:39.080455490Z" level=debug msg="remote introspection plugin filters" filters="[type==io.containerd.runtime.v1 type==io.containerd.runtime.v2]"
time="2022-10-18T12:47:39.087912208Z" level=debug msg="Registering routers"
time="2022-10-18T12:47:39.087941413Z" level=debug msg="Registering GET, /containers/{name:.*}/checkpoints"
time="2022-10-18T12:47:39.088028416Z" level=debug msg="Registering POST, /containers/{name:.*}/checkpoints"
time="2022-10-18T12:47:39.088081666Z" level=debug msg="Registering DELETE, /containers/{name}/checkpoints/{checkpoint}"
time="2022-10-18T12:47:39.088151957Z" level=debug msg="Registering HEAD, /containers/{name:.*}/archive"
time="2022-10-18T12:47:39.088205758Z" level=debug msg="Registering GET, /containers/json"
time="2022-10-18T12:47:39.088246665Z" level=debug msg="Registering GET, /containers/{name:.*}/export"
time="2022-10-18T12:47:39.088289605Z" level=debug msg="Registering GET, /containers/{name:.*}/changes"
time="2022-10-18T12:47:39.088348466Z" level=debug msg="Registering GET, /containers/{name:.*}/json"
time="2022-10-18T12:47:39.088388861Z" level=debug msg="Registering GET, /containers/{name:.*}/top"
time="2022-10-18T12:47:39.088420511Z" level=debug msg="Registering GET, /containers/{name:.*}/logs"
time="2022-10-18T12:47:39.088461447Z" level=debug msg="Registering GET, /containers/{name:.*}/stats"
time="2022-10-18T12:47:39.088485612Z" level=debug msg="Registering GET, /containers/{name:.*}/attach/ws"
time="2022-10-18T12:47:39.088538812Z" level=debug msg="Registering GET, /exec/{id:.*}/json"
time="2022-10-18T12:47:39.088584388Z" level=debug msg="Registering GET, /containers/{name:.*}/archive"
time="2022-10-18T12:47:39.088610086Z" level=debug msg="Registering POST, /containers/create"
time="2022-10-18T12:47:39.088635754Z" level=debug msg="Registering POST, /containers/{name:.*}/kill"
time="2022-10-18T12:47:39.088703410Z" level=debug msg="Registering POST, /containers/{name:.*}/pause"
time="2022-10-18T12:47:39.088751390Z" level=debug msg="Registering POST, /containers/{name:.*}/unpause"
time="2022-10-18T12:47:39.088828044Z" level=debug msg="Registering POST, /containers/{name:.*}/restart"
time="2022-10-18T12:47:39.088874721Z" level=debug msg="Registering POST, /containers/{name:.*}/start"
time="2022-10-18T12:47:39.088913995Z" level=debug msg="Registering POST, /containers/{name:.*}/stop"
time="2022-10-18T12:47:39.088967956Z" level=debug msg="Registering POST, /containers/{name:.*}/wait"
time="2022-10-18T12:47:39.089024602Z" level=debug msg="Registering POST, /containers/{name:.*}/resize"
time="2022-10-18T12:47:39.089067292Z" level=debug msg="Registering POST, /containers/{name:.*}/attach"
time="2022-10-18T12:47:39.089107457Z" level=debug msg="Registering POST, /containers/{name:.*}/copy"
time="2022-10-18T12:47:39.089149927Z" level=debug msg="Registering POST, /containers/{name:.*}/exec"
time="2022-10-18T12:47:39.089190403Z" level=debug msg="Registering POST, /exec/{name:.*}/start"
time="2022-10-18T12:47:39.089221812Z" level=debug msg="Registering POST, /exec/{name:.*}/resize"
time="2022-10-18T12:47:39.089250155Z" level=debug msg="Registering POST, /containers/{name:.*}/rename"
time="2022-10-18T12:47:39.089325115Z" level=debug msg="Registering POST, /containers/{name:.*}/update"
time="2022-10-18T12:47:39.089373305Z" level=debug msg="Registering POST, /containers/prune"
time="2022-10-18T12:47:39.089411647Z" level=debug msg="Registering POST, /commit"
time="2022-10-18T12:47:39.089443206Z" level=debug msg="Registering PUT, /containers/{name:.*}/archive"
time="2022-10-18T12:47:39.089483001Z" level=debug msg="Registering DELETE, /containers/{name:.*}"
time="2022-10-18T12:47:39.089518497Z" level=debug msg="Registering GET, /images/json"
time="2022-10-18T12:47:39.089549826Z" level=debug msg="Registering GET, /images/search"
time="2022-10-18T12:47:39.089574602Z" level=debug msg="Registering GET, /images/get"
time="2022-10-18T12:47:39.089613836Z" level=debug msg="Registering GET, /images/{name:.*}/get"
time="2022-10-18T12:47:39.089641337Z" level=debug msg="Registering GET, /images/{name:.*}/history"
time="2022-10-18T12:47:39.089699506Z" level=debug msg="Registering GET, /images/{name:.*}/json"
time="2022-10-18T12:47:39.089743028Z" level=debug msg="Registering POST, /images/load"
time="2022-10-18T12:47:39.089774306Z" level=debug msg="Registering POST, /images/create"
time="2022-10-18T12:47:39.089837765Z" level=debug msg="Registering POST, /images/{name:.*}/push"
time="2022-10-18T12:47:39.089888540Z" level=debug msg="Registering POST, /images/{name:.*}/tag"
time="2022-10-18T12:47:39.089919018Z" level=debug msg="Registering POST, /images/prune"
time="2022-10-18T12:47:39.089940868Z" level=debug msg="Registering DELETE, /images/{name:.*}"
time="2022-10-18T12:47:39.089985823Z" level=debug msg="Registering OPTIONS, /{anyroute:.*}"
time="2022-10-18T12:47:39.090021510Z" level=debug msg="Registering GET, /_ping"
time="2022-10-18T12:47:39.090041036Z" level=debug msg="Registering HEAD, /_ping"
time="2022-10-18T12:47:39.090075050Z" level=debug msg="Registering GET, /events"
time="2022-10-18T12:47:39.090091330Z" level=debug msg="Registering GET, /info"
time="2022-10-18T12:47:39.090113682Z" level=debug msg="Registering GET, /version"
time="2022-10-18T12:47:39.090127859Z" level=debug msg="Registering GET, /system/df"
time="2022-10-18T12:47:39.090150050Z" level=debug msg="Registering POST, /auth"
time="2022-10-18T12:47:39.090175779Z" level=debug msg="Registering GET, /volumes"
time="2022-10-18T12:47:39.090196267Z" level=debug msg="Registering GET, /volumes/{name:.*}"
time="2022-10-18T12:47:39.090231443Z" level=debug msg="Registering POST, /volumes/create"
time="2022-10-18T12:47:39.090249477Z" level=debug msg="Registering POST, /volumes/prune"
time="2022-10-18T12:47:39.090283090Z" level=debug msg="Registering PUT, /volumes/{name:.*}"
time="2022-10-18T12:47:39.090334766Z" level=debug msg="Registering DELETE, /volumes/{name:.*}"
time="2022-10-18T12:47:39.090377206Z" level=debug msg="Registering POST, /build"
time="2022-10-18T12:47:39.090394608Z" level=debug msg="Registering POST, /build/prune"
time="2022-10-18T12:47:39.090425556Z" level=debug msg="Registering POST, /build/cancel"
time="2022-10-18T12:47:39.090440534Z" level=debug msg="Registering POST, /session"
time="2022-10-18T12:47:39.090460211Z" level=debug msg="Registering POST, /swarm/init"
time="2022-10-18T12:47:39.090490688Z" level=debug msg="Registering POST, /swarm/join"
time="2022-10-18T12:47:39.090506007Z" level=debug msg="Registering POST, /swarm/leave"
time="2022-10-18T12:47:39.090526536Z" level=debug msg="Registering GET, /swarm"
time="2022-10-18T12:47:39.090551462Z" level=debug msg="Registering GET, /swarm/unlockkey"
time="2022-10-18T12:47:39.090567973Z" level=debug msg="Registering POST, /swarm/update"
time="2022-10-18T12:47:39.090593611Z" level=debug msg="Registering POST, /swarm/unlock"
time="2022-10-18T12:47:39.090623848Z" level=debug msg="Registering GET, /services"
time="2022-10-18T12:47:39.090638525Z" level=debug msg="Registering GET, /services/{id}"
time="2022-10-18T12:47:39.090708657Z" level=debug msg="Registering POST, /services/create"
time="2022-10-18T12:47:39.090737751Z" level=debug msg="Registering POST, /services/{id}/update"
time="2022-10-18T12:47:39.090765253Z" level=debug msg="Registering DELETE, /services/{id}"
time="2022-10-18T12:47:39.090849601Z" level=debug msg="Registering GET, /services/{id}/logs"
time="2022-10-18T12:47:39.090893854Z" level=debug msg="Registering GET, /nodes"
time="2022-10-18T12:47:39.090920333Z" level=debug msg="Registering GET, /nodes/{id}"
time="2022-10-18T12:47:39.090943386Z" level=debug msg="Registering DELETE, /nodes/{id}"
time="2022-10-18T12:47:39.090971419Z" level=debug msg="Registering POST, /nodes/{id}/update"
time="2022-10-18T12:47:39.091008228Z" level=debug msg="Registering GET, /tasks"
time="2022-10-18T12:47:39.091032714Z" level=debug msg="Registering GET, /tasks/{id}"
time="2022-10-18T12:47:39.091064503Z" level=debug msg="Registering GET, /tasks/{id}/logs"
time="2022-10-18T12:47:39.091094690Z" level=debug msg="Registering GET, /secrets"
time="2022-10-18T12:47:39.091123574Z" level=debug msg="Registering POST, /secrets/create"
time="2022-10-18T12:47:39.091144122Z" level=debug msg="Registering DELETE, /secrets/{id}"
time="2022-10-18T12:47:39.091177645Z" level=debug msg="Registering GET, /secrets/{id}"
time="2022-10-18T12:47:39.091198063Z" level=debug msg="Registering POST, /secrets/{id}/update"
time="2022-10-18T12:47:39.091224804Z" level=debug msg="Registering GET, /configs"
time="2022-10-18T12:47:39.091256623Z" level=debug msg="Registering POST, /configs/create"
time="2022-10-18T12:47:39.091272142Z" level=debug msg="Registering DELETE, /configs/{id}"
time="2022-10-18T12:47:39.091309272Z" level=debug msg="Registering GET, /configs/{id}"
time="2022-10-18T12:47:39.091342955Z" level=debug msg="Registering POST, /configs/{id}/update"
time="2022-10-18T12:47:39.091388180Z" level=debug msg="Registering GET, /plugins"
time="2022-10-18T12:47:39.091416974Z" level=debug msg="Registering GET, /plugins/{name:.*}/json"
time="2022-10-18T12:47:39.091442411Z" level=debug msg="Registering GET, /plugins/privileges"
time="2022-10-18T12:47:39.091470624Z" level=debug msg="Registering DELETE, /plugins/{name:.*}"
time="2022-10-18T12:47:39.091497815Z" level=debug msg="Registering POST, /plugins/{name:.*}/enable"
time="2022-10-18T12:47:39.091537640Z" level=debug msg="Registering POST, /plugins/{name:.*}/disable"
time="2022-10-18T12:47:39.091570291Z" level=debug msg="Registering POST, /plugins/pull"
time="2022-10-18T12:47:39.091601359Z" level=debug msg="Registering POST, /plugins/{name:.*}/push"
time="2022-10-18T12:47:39.091631025Z" level=debug msg="Registering POST, /plugins/{name:.*}/upgrade"
time="2022-10-18T12:47:39.091690827Z" level=debug msg="Registering POST, /plugins/{name:.*}/set"
time="2022-10-18T12:47:39.091729158Z" level=debug msg="Registering POST, /plugins/create"
time="2022-10-18T12:47:39.091758914Z" level=debug msg="Registering GET, /distribution/{name:.*}/json"
time="2022-10-18T12:47:39.091841819Z" level=debug msg="Registering POST, /grpc"
time="2022-10-18T12:47:39.091872547Z" level=debug msg="Registering GET, /networks"
time="2022-10-18T12:47:39.091888858Z" level=debug msg="Registering GET, /networks/"
time="2022-10-18T12:47:39.091918754Z" level=debug msg="Registering GET, /networks/{id:.+}"
time="2022-10-18T12:47:39.091957005Z" level=debug msg="Registering POST, /networks/create"
time="2022-10-18T12:47:39.091975229Z" level=debug msg="Registering POST, /networks/{id:.*}/connect"
time="2022-10-18T12:47:39.092010385Z" level=debug msg="Registering POST, /networks/{id:.*}/disconnect"
time="2022-10-18T12:47:39.092033298Z" level=debug msg="Registering POST, /networks/prune"
time="2022-10-18T12:47:39.092062162Z" level=debug msg="Registering DELETE, /networks/{id:.*}"
time="2022-10-18T12:47:39.092184000Z" level=info msg="API listen on /tmp/integration4180081475/dww2uhqidy16j/docker.sock"
time="2022-10-18T12:47:39.092361813Z" level=debug msg="Calling HEAD /_ping"
time="2022-10-18T12:47:39.119859763Z" level=debug msg="Calling POST /grpc"
time="2022-10-18T12:47:39.120499180Z" level=debug msg="session started"
time="2022-10-18T12:47:39.120562779Z" level=info msg="[core] original dial target is: \"localhost\"" module=grpc
time="2022-10-18T12:47:39.120614686Z" level=info msg="[core] parsed dial target is: {Scheme: Authority: Endpoint:localhost URL:{Scheme: Opaque: User: Host: Path:localhost RawPath: ForceQuery:false RawQuery: Fragment: RawFragment:}}" module=grpc  
time="2022-10-18T12:47:39.120622340Z" level=info msg="[core] fallback to scheme \"passthrough\"" module=grpc
time="2022-10-18T12:47:39.120634784Z" level=info msg="[core] parsed dial target is: {Scheme:passthrough Authority: Endpoint:localhost URL:{Scheme:passthrough Opaque: User: Host: Path:/localhost RawPath: ForceQuery:false RawQuery: Fragment: RawFragment:}}" module=grpc
time="2022-10-18T12:47:39.120640835Z" level=info msg="[core] Channel authority set to \"localhost\"" module=grpc
time="2022-10-18T12:47:39.120655302Z" level=info msg="[core] ccResolverWrapper: sending update to cc: {[{localhost  <nil> <nil> 0 <nil>}] <nil> <nil>}" module=grpc
time="2022-10-18T12:47:39.120675670Z" level=info msg="[core] ClientConn switching balancer to \"pick_first\"" module=grpc
time="2022-10-18T12:47:39.120681521Z" level=info msg="[core] Channel switches to new LB policy \"pick_first\"" module=grpc
time="2022-10-18T12:47:39.120736494Z" level=info msg="[core] Subchannel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:39.120759948Z" level=info msg="[core] Subchannel picks a new address \"localhost\" to connect" module=grpc
time="2022-10-18T12:47:39.120947656Z" level=info msg="[core] Channel Connectivity change to CONNECTING" module=grpc
time="2022-10-18T12:47:39.121327111Z" level=info msg="[core] Subchannel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:39.121358069Z" level=info msg="[core] Channel Connectivity change to READY" module=grpc
time="2022-10-18T12:47:39.121513993Z" level=debug msg="skipping invalid cache export mode: "
time="2022-10-18T12:47:39.134853019Z" level=debug msg=resolving host=registry-1.docker.io
time="2022-10-18T12:47:39.134898484Z" level=debug msg="do request" host=registry-1.docker.io request.header.accept="application/vnd.docker.distribution.manifest.v2+json, application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.oci.image.manifest.v1+json, application/vnd.oci.image.index.v1+json, */*" request.header.user-agent=buildkit/0.0.0+unknown request.method=HEAD url="https://registry-1.docker.io/v2/buildkit_test/pnkfrjl08i2qpegzaj4mvxww6/manifests/latest"
time="2022-10-18T12:47:39.533106074Z" level=debug msg="fetch response received" host=registry-1.docker.io response.header.content-length=182 response.header.content-type=application/json response.header.date="Tue, 18 Oct 2022 12:47:41 GMT" response.header.docker-distribution-api-version=registry/2.0 response.header.docker-ratelimit-source=xx.xx.xx.xx response.header.strict-transport-security="max-age=31536000" response.header.www-authenticate="Bearer realm=\"https://auth.docker.io/token\",service=\"registry.docker.io\",scope=\"repository:buildkit_test/pnkfrjl08i2qpegzaj4mvxww6:pull\"" response.status="401 Unauthorized" url="https://registry-1.docker.io/v2/buildkit_test/pnkfrjl08i2qpegzaj4mvxww6/manifests/latest"
time="2022-10-18T12:47:39.533157912Z" level=debug msg=Unauthorized header="Bearer realm=\"https://auth.docker.io/token\",service=\"registry.docker.io\",scope=\"repository:buildkit_test/pnkfrjl08i2qpegzaj4mvxww6:pull\"" host=registry-1.docker.io  
time="2022-10-18T12:47:39.534715740Z" level=debug msg="do request" host=registry-1.docker.io request.header.accept="application/vnd.docker.distribution.manifest.v2+json, application/vnd.docker.distribution.manifest.list.v2+json, application/vnd.oci.image.manifest.v1+json, application/vnd.oci.image.index.v1+json, */*" request.header.user-agent=buildkit/0.0.0+unknown request.method=HEAD url="https://registry-1.docker.io/v2/buildkit_test/pnkfrjl08i2qpegzaj4mvxww6/manifests/latest"
time="2022-10-18T12:47:40.151977938Z" level=debug msg="fetch response received" host=registry-1.docker.io response.header.content-length=182 response.header.content-type=application/json response.header.date="Tue, 18 Oct 2022 12:47:41 GMT" response.header.docker-distribution-api-version=registry/2.0 response.header.docker-ratelimit-source=xx.xx.xx.xx response.header.strict-transport-security="max-age=31536000" response.header.www-authenticate="Bearer realm=\"https://auth.docker.io/token\",service=\"registry.docker.io\",scope=\"repository:buildkit_test/pnkfrjl08i2qpegzaj4mvxww6:pull\",error=\"insufficient_scope\"" response.status="401 Unauthorized" url="https://registry-1.docker.io/v2/buildkit_test/pnkfrjl08i2qpegzaj4mvxww6/manifests/latest"
time="2022-10-18T12:47:40.152028012Z" level=debug msg=Unauthorized header="Bearer realm=\"https://auth.docker.io/token\",service=\"registry.docker.io\",scope=\"repository:buildkit_test/pnkfrjl08i2qpegzaj4mvxww6:pull\",error=\"insufficient_scope\"" host=registry-1.docker.io
time="2022-10-18T12:47:40.152495578Z" level=info msg="trying next host" error="pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed" host=registry-1.docker.io
time="2022-10-18T12:47:40.154803040Z" level=info msg="[core] Subchannel Connectivity change to IDLE" module=grpc
time="2022-10-18T12:47:40.154856460Z" level=debug msg="session finished: <nil>"
time="2022-10-18T12:47:40.154869254Z" level=info msg="[core] Channel Connectivity change to IDLE" module=grpc
time="2022-10-18T12:47:40.154892928Z" level=info msg="[core] Channel Connectivity change to SHUTDOWN" module=grpc
time="2022-10-18T12:47:40.154899942Z" level=info msg="[core] Subchannel Connectivity change to SHUTDOWN" module=grpc
    dockerfile_test.go:3954:
                Error Trace:    dockerfile_test.go:3954
                                                        run.go:86
                                                        run.go:189
                Error:          Received unexpected error:
                                pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
                                github.com/moby/buildkit/util/stack.Enable
                                        /src/util/stack/stack.go:77
                                github.com/moby/buildkit/util/grpcerrors.FromGRPC
                                        /src/util/grpcerrors/grpcerrors.go:196
                                github.com/moby/buildkit/util/grpcerrors.UnaryClientInterceptor
                                        /src/util/grpcerrors/intercept.go:41
                                google.golang.org/grpc.(*ClientConn).Invoke
                                        /src/vendor/google.golang.org/grpc/call.go:35
                                github.com/moby/buildkit/api/services/control.(*controlClient).Solve
                                        /src/api/services/control/control.pb.go:1535
                                github.com/moby/buildkit/client.(*Client).solve.func2
                                        /src/client/solve.go:231
                                golang.org/x/sync/errgroup.(*Group).Go.func1
                                        /src/vendor/golang.org/x/sync/errgroup/errgroup.go:57
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1594
                                failed to solve
                                github.com/moby/buildkit/client.(*Client).solve.func2
                                        /src/client/solve.go:244
                                golang.org/x/sync/errgroup.(*Group).Go.func1
                                        /src/vendor/golang.org/x/sync/errgroup/errgroup.go:57
                                runtime.goexit
                                        /usr/local/go/src/runtime/asm_amd64.s:1594
                Test:           TestIntegration/TestCacheMultiPlatformImportExport/worker=dockerd-containerd/frontend=gateway
> sending sigterm 2022-10-18 12:47:40.15589221 +0000 UTC m=+5.683423504
time="2022-10-18T12:47:40.156005321Z" level=info msg="Processing signal 'terminated'"
time="2022-10-18T12:47:40.156101711Z" level=debug msg="daemon configured with a 15 seconds minimum shutdown timeout"
time="2022-10-18T12:47:40.156124594Z" level=debug msg="start clean shutdown of all containers with a 15 seconds timeout..."
time="2022-10-18T12:47:40.156432691Z" level=debug msg="Unix socket /tmp/dxr/dww2uhqidy16j/libnetwork/8c96b0830f42.sock doesn't exist. cannot accept client connections"
time="2022-10-18T12:47:40.156459111Z" level=info msg="[core] Channel Connectivity change to SHUTDOWN" module=grpc
time="2022-10-18T12:47:40.156468629Z" level=info msg="[core] Subchannel Connectivity change to SHUTDOWN" module=grpc
time="2022-10-18T12:47:40.156474991Z" level=debug msg="Cleaning up old mountid : start."
time="2022-10-18T12:47:40.156536066Z" level=info msg="stopping event stream following graceful shutdown" error="<nil>" module=libcontainerd namespace=dww2uhqidy16j
time="2022-10-18T12:47:40.156643075Z" level=debug msg="Cleaning up old mountid : done."
time="2022-10-18T12:47:40.156767609Z" level=debug msg="Clean shutdown succeeded"
time="2022-10-18T12:47:40.156781986Z" level=info msg="Daemon shutdown complete"
time="2022-10-18T12:47:40.156831079Z" level=info msg="stopping healthcheck following graceful shutdown" module=libcontainerd
time="2022-10-18T12:47:40.156860303Z" level=info msg="[core] Channel Connectivity change to SHUTDOWN" module=grpc
time="2022-10-18T12:47:40.156873749Z" level=info msg="[core] Subchannel Connectivity change to SHUTDOWN" module=grpc
time="2022-10-18T12:47:40.156988505Z" level=debug msg="received signal" signal=terminated
time="2022-10-18T12:47:40.157079836Z" level=debug msg="sd notification" error="<nil>" notified=false state="STOPPING=1"
> stopped 2022-10-18 12:47:40.158855757 +0000 UTC m=+5.686387051 exit status 0 0
--- FAIL: TestIntegration (4.23s)
    --- FAIL: TestIntegration/TestCacheMultiPlatformImportExport/worker=dockerd-containerd/frontend=gateway (1.45s)
FAIL
FAIL    github.com/moby/buildkit/frontend/dockerfile    5.691s
FAIL
```